src/tests/HDFFile.cxx

```diff
@@ -7,10 +7,33 @@
 #include "../helper.h"
```

Comment:
This file has the same name as the implementation it is testing, consider adding "Test" to the name.